tests/sentry/replays/test_project_replay_recording_segment_index.py

```diff
@@ -0,0 +1,55 @@
+from django.urls import reverse
+
+from sentry.models import File
+from sentry.replays.models import ReplayRecordingSegment
+from sentry.testutils import APITestCase
+
+
+class ProjectReplayRecordingSegmentTestCase(APITestCase):
+    endpoint = "sentry-api-0-project-replay-recording-segment-index"
+
+    def setUp(self):
+        super().setUp()
+
+        self.replay_id = "6f959c5c-bc77-4683-8723-6e3367b0cfac"
```

Comment:
could just remove the dashes here so you don't have to replace below, or just do `replay_id = uuid.uuid4().hex`